build.sbt

```diff
@@ -57,7 +57,8 @@ lazy val commonSettings = Seq(
     "-unchecked",
   ),
   javacOptions ++= Seq(
-    "-Werror",
+    "-source", "17",
+//    "-Werror",
```

Comment:
Do we have some warnings? Please silence them with annotations if needed, and re-enable this.

Reply:
impossible. its coming from <3 protobuf <3

Reply:
annnd i seem to be unable to disable deprecation selectively on it..